Automattic/woocommerce-payments
# Pull in and use the e2e-utils and api packages for e2e tests

Fixes https://github.com/Automattic/woocommerce-payments/issues/1621

#### Changes proposed in this Pull Request

This PR pulls in both the `@woocommerce/api` and `@woocommerce/e2e-utils` packages and updates existing tests and workflows to use these packages. As a result of bringing these in the following changes are also proposed:

* Remove the `StoreOwnerFlow` as this is covered with the `merchant` flows from the `@woocommerce/e2e-utils`.
* Removes the `helper.js` file as these helper functions also exist in `@woocommerce/e2e-utils`.
* Uses the `shopper` flows included in `@woocommerce/e2e-utils` for shared functionality.
* Renames `PaymentsCustomerFlow` to `paymentsShopper` to be more aligned with the `shopper` naming convention in `@woocommerce/e2e-utils`.
* Update tests to use `paymentsShopper` and `shopper` flows.

Please see paJDYF-1m9-p2 for more context on these changes.

#### Testing instructions

* Make sure the CI tests pass
* Run the full e2e test suite, either in headless (`npm run test:e2e`) or non-headless mode (`npm run test:e2e-dev`) and verify the tests pass
-------------------
